src/main/java/io/mathdojo/useraccountservice/AccountRequestBodyUsersHandler.java

```diff
@@ -38,7 +38,7 @@ public HttpResponseMessage executePostForNewUserInOrg(
         try {
             AccountModificationRequest requestBody = request.getBody().get();
             AccountModificationRequest modificationRequest = new AccountModificationRequest(
-                null, orgId, requestBody.isAccountVerified(), requestBody.getName(), requestBody.getProfileImageLink()
+            		requestBody.getId(), orgId, requestBody.isAccountVerified(), requestBody.getName(), requestBody.getProfileImageLink()
```

Comment:
how come you had to change this again, wasn't this done in this commit: 
https://github.com/math-dojo/user-account-service/pull/42/commits/97bf8cb9824e6696f61a9f3af7fbd992d4dbe880#diff-d14ea632c2c8172184c28cc434af3c7995c0977a474c46ce88f7ba7275c787e6R35